src/config/CodeGeneratorConfig.php

```diff
@@ -4,9 +4,9 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class CodeGeneratorConfig {
-	
+
 	protected $options;
-	
+
 	public function __construct(array $options = []) {
```

Comment:
That is a good point. The problem is how this strict types stuff actually works. It also applies to functions called in another file and it does not specifically require that you specify types (just that if you specify them the values passed through them have to be valid even for scalar types).